### ISSUE TITLE
Auth 0 Dev Keys

### ISSUE THREAD

**Issue by Skillkiller** [zekroTJA/pronote#4]
A warning is displayed on the login page for Auth0 that one or more connections are using development keys.

![Screenshot 2024-08-04 at 23-20-37 Log in Pronote](https://github.com/user-attachments/assets/4e887b63-ddcd-4b6f-8a43-66e02384dde8)

Is this done on purpose because it is a test environment?

**Comment by zekroTJA** [comment]
Yeah, that's because the Google Connection still uses Auth0's keys instead of a custom created OAuth2 app like with the GitHub connection.

I wanted to go through creating an OAuth2 App on Google for that, but it includes a reviewing process which I was too lazy to do that right now. I'll probably switch the keys when I find the time and motivation to create a review for the app, but for now, I would leave it as is, because some people already singed in with google which would render their accounts unusable when I would remove the login option.
